tests/test_extractors/test_flashtext_entity_extractor.py

```diff
@@ -0,0 +1,234 @@
+from typing import Text, List, Dict, Any
+
```

Comment:
Could you add a smoketest? I've got some Rasa pipelines with the component in there. The goal of these tests is to run the most basic end-to-end check by running a specific `config.yml` file that contains the component. It's not meant as a well-understood test, but if something is wrong then it is a useful signal for further investigation.